<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptionsDialog</class>
 <widget class="QDialog" name="DisplayOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Point Size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="pointSizeSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pointSizeSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Point Density</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="pointDensitySpinBox">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pointDensitySlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="smoothPointCheckBox">
     <property name="text">
      <string>Smooth Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="colorCheckBox">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="depthMaskCheckBox">
     <property name="text">
      <string>Depth Mask</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="multisampleCheckBox">
     <property name="text">
      <string>Multisampling</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pointSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointDensitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointDensitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointDensitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pointDensitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
